commands/releases/output.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+let cli = require('heroku-cli-util')
+let co = require('co')
+
+function * run (context, heroku) {
+  function getLatestRelease () {
+    return heroku.request({
+      path: `/apps/${context.app}/releases`,
+      partial: true,
+      headers: { 'Range': 'version ..; max=1, order=desc' }
+    }).then((releases) => releases[0])
+  }
+
+  let release
+  if (context.args.release) {
+    let id = context.args.release.toLowerCase()
+    id = id.startsWith('v') ? id.slice(1) : id
+    release = yield heroku.get(`/apps/${context.app}/releases/${id}`)
+  } else {
+    release = yield getLatestRelease()
+  }
+  let streamUrl = release.output_stream_url
+
+  if (!streamUrl) {
+    cli.warn(`Release v${release.version} has no release output available.`)
+    return
+  }
+
+  yield new Promise(function (resolve, reject) {
+    cli.got.stream(streamUrl)
+      .on('error', reject)
+      .on('end', resolve)
+      .on('data', function (c) {
+        cli.log(c.toString('utf8'))
```

Comment:
It could definitely happen, for any case where the process sends no output for a few seconds. So I think it's something we need to fix before we ship this pr.